libs/judicial-system/types/src/lib/case.ts

```diff
@@ -247,6 +249,24 @@ export const isCompletedCase = (state?: CaseState | null): boolean => {
   return Boolean(state && completedCaseStates.includes(state))
 }
 
+export const isTrafficViolationCase = (
```

Comment:
Consider calling this function from the stepHelper instead of having the logic in 2 places, and then just add the extra CaseFileCategory.Indictment check in to the stepHelper function. Also consider renaming either one so it's clear what the difference is

Reply:
Do you mean moving this function to the stepHelper and calling it from there? The stepHelper file is in under `web` so I can't call a function in there from the backend. Am I misunderstanding?

Reply:
No I mean calling this function from the stepHelper and removing that logic from the isTrafficViolationCase there (and renaming it as well), so the stepHelper function would be something like 

`export const isTrafficViolationCaseCheck = (workingCase: Case): boolean => {
  const isTrafficViolation = isTrafficViolationCase(
    workingCase.indictmentSubtypes,
    workingCase.type as CaseType,
  )
  return Boolean(
    isTrafficViolation &&
      !(
        workingCase.caseFiles &&
        workingCase.caseFiles.find(
          (file) => file.category === CaseFileCategory.INDICTMENT,
        )
      ),
  )
}`
(except maybe with a better name)
My thinking is it might get confusing down the line to have 2 different functions that have the same name and do almost the same thing, instead of reusing the shared logic from the libs function and then adding the missing piece to the stepHelper one.

Reply:
Ah, gotcha ;) 

Onit..